.github/workflows/python-integration-tests.yml

```diff
@@ -233,12 +264,18 @@ jobs:
         working-directory: python
     runs-on: ${{ matrix.os }}
     environment: "integration"
+    env:
+      HNSWLIB_NO_NATIVE: 1
+      Python_Integration_Tests: Python_Integration_Tests
```

Comment:
These two env variables are already specified in the workflow env variables list.